ChangeLog

```diff
@@ -7,6 +7,7 @@ Revision history for Rex
  - Detect invalid hostgroup expressions
 
  [DOCUMENTATION]
+ - Correct type of variable shown as being returned from 'get cmdb'.
```

Comment:
Other changelog entries are simple list entries and do not end with a period.

Reply:
Done.